Scripts/IncidentFields/IncidentFields.py

```diff
@@ -0,0 +1,77 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def get_field_by_long_name(fields, name):
+    for field in fields:
+        if field['name'] == name:
+            return field
+    return None
+
+
+def get_field_by_short_name(fields, name):
+    for field in fields:
+        if field['cliName'] == name:
+            return field
+    return None
+
+
+def get_short_name(value):
+    return re.sub(r'[^A-Za-z0-9]', '', value).lower()
```

Comment:
this function isn't used as well

Reply:
Yep, leftover from an old version and now removed